sagemaker-debugger/tensorflow_action_on_rule/detect_stalled_training_job_and_stop.ipynb

```diff
@@ -4,30 +4,24 @@
    "cell_type": "markdown",
```

Comment:
<p><strong>1) </strong></p><p>"This notebook guides you how to use the <code>StalledTrainingRule</code> built-in rule." needs to be reworded.</p><p> </p><p>Example:</p><p>"This notebook shows you how to use the <code>StalledTrainingRule</code> built-in rule."</p><p> </p><p><strong>2) </strong></p><p>In this sentence "This functionality helps you monitor the training job status and save redundant resource usage." did you mean to say "This functionality helps you monitor the training job status and <u>reduces</u> redundant resource usage."</p><p> </p><p><strong>3)</strong></p><p>The sentence: "If you want to explore using the manual hook registration, " --&gt; if you want to explore <em>what</em>? Maybe "If you want to use debugger with..."?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1592/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Feedback incorporated